### DISCUSSION TITLE
关于更新后的PPT要求

### DISCUSSION THREAD

**Discussion by huanhuan6666** [liusy58/NJU_AdvancedProgramming_SP22#76] [Q&A] [answered]
可以使用全英文，是不是意味着完全不用考虑中文字符的问题了，比如还的判断占3个字符2个字符等等，而是直接.length()就完事

**Comment by Price1999a** [comment, marked as answer]
是的

**Reply by huanhuan6666** [reply]
是否文件表头和SQL指令也必须变成英文，比如SET 商品状态 = 已下架变成 SET state = unsale。

还是说不用做过多改变，在内部的表示可以按照原来的中文约定，即文件表头和SQL指令仍然可以出现中文，只是输入的测试用例全变成了英文。

**Reply by binbin00** [reply]
> 是否文件表头和SQL指令也必须变成英文，比如SET 商品状态 = 已下架变成 SET state = unsale。
> 
> 还是说不用做过多改变，在内部的表示可以按照原来的中文约定，即文件表头和SQL指令仍然可以出现中文，只是输入的测试用例全变成了英文。
请下载最新版的文档和参考文件。

**Reply by WT1W** [reply]
> 更新后的英文commands文件只给了SELECT * FROM commodity的情况，其他情况的英文格式不清楚

**Reply by huanhuan6666** [reply]
这个命令和中文版的没有区别，我觉得更新说明可能只是单纯简化中文字符的判断问题，剩下的可以完全不做改动

**Reply by binbin00** [reply]
> 这个命令和中文版的没有区别，我觉得更新说明可能只是单纯简化中文字符的判断问题，剩下的可以完全不做改动

文件里存的是什么，那命令里就应该是什么。如果存的中文，那就参考ppt中的例子，如果存的英文，那就替换一下就可以了。此版ppt的更新最主要是 允许全英文 的实现，可以避免对中文字符的处理。
